components/support/base/src/test/java/mozilla/components/support/base/facts/FactsTest.kt

```diff
@@ -0,0 +1,59 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.support.base.facts
+
+import mozilla.components.support.base.Component
+import mozilla.components.support.test.mock
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.mockito.Mockito.verify
+
+class FactsTest {
+    @Before
+    @After
+    fun cleanUp() {
+        Facts.clearProcessors()
+    }
+
+    @Test
+    fun `Collecting fact without processor`() {
+        Facts.collect(Fact(
```

Comment:
🤔 hm there's no assertion here. I guess we want to verify that this is a NOOP an doesn't throw an exception?

Reply:
Not sure how do check that something is a NOOP here though? Running it at least checks that there's no exception - that would fail the test. :)